internal/functions/efgs/api/structs.go

```diff
@@ -93,22 +93,22 @@ type BatchImportParams struct {
 
 //DiagnosisKeyWrapper map json response from EFGS to local DiagnosisKey structure
 type DiagnosisKeyWrapper struct {
-	tableName                  struct{}  `pg:"diagnosis_keys,alias:dk"`
-	ID                         int32     `pg:",pk" json:"id"`
-	CreatedAt                  time.Time `pg:"default:now()" json:"created_at"`
-	KeyData                    string    `pg:",notnull,unique" json:"keyData,omitempty"`
-	RollingStartIntervalNumber uint32    `pg:",use_zero" json:"rollingStartIntervalNumber,omitempty"`
-	RollingPeriod              uint32    `pg:",use_zero" json:"rollingPeriod,omitempty"`
-	TransmissionRiskLevel      int32     `pg:",use_zero" json:"transmissionRiskLevel,omitempty"`
-	VisitedCountries           []string  `json:"visitedCountries,omitempty"`
-	Origin                     string    `pg:"default:'CZ'" json:"origin,omitempty"`
-	ReportType                 int       `pg:",use_zero" json:"reportType,omitempty"`
-	DaysSinceOnsetOfSymptoms   int32     `pg:",use_zero" json:"days_since_onset_of_symptoms,omitempty"`
-	Retries                    int       `pg:"default:0,use_zero" json:"retries,omitempty"`
-	IsUploaded                 bool      `pg:"default:False,notnull,use_zero" json:"isUploaded,omitempty"`
+	tableName                  struct{}   `pg:"diagnosis_keys,alias:dk"`
+	ID                         int32      `pg:",pk" json:"id"`
+	CreatedAt                  time.Time  `pg:"default:now()" json:"created_at"`
+	KeyData                    string     `pg:",notnull,unique" json:"keyData,omitempty"`
+	RollingStartIntervalNumber uint32     `pg:",use_zero" json:"rollingStartIntervalNumber,omitempty"`
+	RollingPeriod              uint32     `pg:",use_zero" json:"rollingPeriod,omitempty"`
+	TransmissionRiskLevel      int32      `pg:",use_zero" json:"transmissionRiskLevel,omitempty"`
+	VisitedCountries           []string   `json:"visitedCountries,omitempty"`
+	Origin                     string     `pg:"default:'CZ'" json:"origin,omitempty"`
+	ReportType                 ReportType `pg:",use_zero" json:"reportType,omitempty"`
+	DaysSinceOnsetOfSymptoms   int32      `pg:",use_zero" json:"days_since_onset_of_symptoms,omitempty"`
+	Retries                    int        `pg:"default:0,use_zero" json:"retries,omitempty"`
+	IsUploaded                 bool       `pg:"default:False,notnull,use_zero" json:"isUploaded,omitempty"`
 }
 
-//ToData convert struct from DiagnosisKeyWrapper to DiagnosisKey
+//ToData convert struct from DiagnosisKeyWrapper to DiagnosisKey. Returns nil if DiagnosisKeyWrapper is not valid
 func (wrappedKey *DiagnosisKeyWrapper) ToData() *DiagnosisKey {
```

Comment:
Given we have the function `DiagnosisKey.ToWrappedKey`, shouldn't this be named `ToInnerKey`? Or, `ToWrapper` + `ToData` would be fine for me too (and it would cause less changes in existing code) - it's just inconsistent now...